godel/config/dist-plugin.yml

```diff
@@ -8,8 +8,12 @@ products:
       os-archs:
       - os: darwin
         arch: amd64
+      - os: darwin
+        arch: arm64
       - os: linux
         arch: amd64
+      - os: linux
```

Comment:
This is probably the right thing to do long-term, but would like to scope this PR to just adding darwin-arm64 support